components/popover/popover-mixin.js

```diff
@@ -72,26 +134,37 @@ export const PopoverMixin = superclass => class extends superclass {
 					animation: var(--d2l-popover-animation-name, var(--d2l-popover-default-animation-name)) 300ms ease;
 				}
 			}
+
+			:host([_offscreen]) {
```

Comment:
As with fixed position dropdown, we position offscreen when the opener is scrolled out of view.